pilot/pkg/features/experimental.go

```diff
@@ -198,4 +198,7 @@ var (
 
 	UnifiedSidecarScoping = env.Register("PILOT_UNIFIED_SIDECAR_SCOPE", true,
 		"If true, unified SidecarScope creation will be used. This is only intended as a temporary feature flag for backwards compatibility.").Get()
+
+	EnvoyStatusPortEnableProxyProtocol = env.Register("ENVOY_STATUS_PORT_ENABLE_PROXY_PROTOCOL", false,
```

Comment:
IIUC, this's a flag for pilot-agent, we need a better name or more clear doc to avoid issue in https://github.com/istio/istio/issues/47925.

Reply:
Sure, which name do you suggest? `PILOT_ENVOY_STATUS_PORT_ENABLE_PROXY_PROTOCOL`?

Reply:
@zirain would `PILOT_ENVOY_STATUS_PORT_ENABLE_PROXY_PROTOCOL` be OK for you?

Reply:
Friendly ping @zirain :wink:

Reply:
Hello, sorry, I’m a bit lost: I understood that `ENVOY_STATUS_PORT_ENABLE_PROXY_PROTOCOL` was not suitable because it was not clear that it’s a flag for pilot-agent, so I proposed to add the `PILOT_` prefix, but it’s not suitable either. Which name do you suggest? Is the original name a good one in the end?